sql/report_queries/loans_and_renewals_counts/README.md

```diff
@@ -6,6 +6,8 @@
 
 Brief description: This report can be used to satisfy a need for annual statistics showing total loans and renewals by location. The report itself returns each loan record, with information about the item locations, material type, loan policy, loan type, and loan status. To summarize loans and renewals for different subsets of the data, add up the loans and renewals columns for the subsets.
 
+Note that currently there is no loan renewal date available. 
```

Comment:
Does this refer specifically to the folio-snapshot reference data, or in general?

Reply:
In general, FOLIO does not capture renewal date

Reply:
My mistake; I read it as "loan renewal data."